lib/line/bot/client.rb

```diff
@@ -135,9 +132,6 @@ def issue_channel_access_token_jwt(jwt)
       #
       # @return [Net::HTTPResponse]
       def revoke_channel_access_token_jwt(access_token)
-        channel_id_required
-        channel_secret_required
```

Comment:
I think these two lines are necessary for below code.
https://github.com/line/line-bot-sdk-ruby/pull/240/files#diff-4c889fbe50c5c796640e454b912fc50d15bcb660970c1a6eabd806e5a3a1842fL143-L144

Reply:
Sorry, change back 🙏